scripts/exgdas_global_marine_analysis_vrfy.py

```diff
@@ -41,164 +43,127 @@
+                      variables_horiz={'Temp': [-0.5, 0.5],
+                                       'Salt': [-0.1, 0.1],
+                                       'ave_ssh': [-0.1, 0.1]},
+                      variables_meridional={'Temp': [-0.5, 0.5],
+                                            'Salt': [-0.1, 0.1]},
+                      colormap='RdBu',
+                      comout=os.path.join(comout, 'vrfy', 'incr')),   # ocean increment
+           plotConfig(grid_file=grid_file,
+                      data_file=os.path.join(comout, f'{RUN}.t'+cyc+'z.ice.incr.nc'),
+                      lats=np.arange(-60, 60, 10),
+                      variables_horiz={'aicen': [-0.2, 0.2],
+                                       'hicen': [-0.5, 0.5],
+                                       'hsnon': [-0.1, 0.1]},
+                      colormap='RdBu',
+                      projs=['North', 'South'],
+                      comout=os.path.join(comout, 'vrfy', 'incr')),   # sea ice increment
+           plotConfig(grid_file=grid_file,
+                      data_file=os.path.join(comout, f'{RUN}.t'+cyc+'z.iceana.nc'),
+                      variables_horiz={'aicen': [0.0, 1.0],
+                                       'hicen': [0.0, 4.0],
+                                       'hsnon': [0.0, 0.5]},
+                      colormap='jet',
+                      projs=['North', 'South', 'Global'],
+                      comout=os.path.join(comout, 'vrfy', 'ana')),   # sea ice analysis
+           plotConfig(grid_file=grid_file,
+                      data_file=os.path.join(com_ice_history, f'{RUN}.t{gcyc}z.icef006.nc'),
+                      variables_horiz={'aice_h': [0.0, 1.0],
+                                       'hs_h': [0.0, 4.0],
+                                       'hi_h': [0.0, 0.5]},
+                      colormap='jet',
+                      projs=['North', 'South', 'Global'],
+                      comout=os.path.join(comout, 'vrfy', 'bkg')),   # sea ice background
+           plotConfig(grid_file=grid_file,
+                      data_file=os.path.join(comout, f'{RUN}.t'+cyc+'z.ocnana.nc'),
+                      variables_horiz={'ave_ssh': [-1.8, 1.3],
+                                       'Temp': [-1.8, 34.0],
+                                       'Salt': [30, 38]},
+                      colormap='jet',
+                      comout=os.path.join(comout, 'vrfy', 'ana')),   # ocean surface analysis
+           plotConfig(grid_file=grid_file,
+                      data_file=os.path.join(com_ocean_history, f'{RUN}.t{gcyc}z.ocnf006.nc'),
+                      variables_horiz={'ave_ssh': [-1.8, 1.3],
+                                       'Temp': [-1.8, 34.0],
+                                       'Salt': [30, 38]},
+                      colormap='jet',
+                      comout=os.path.join(comout, 'vrfy', 'bkg'))]   # ocean surface background
+
+# Number of processes (adjust as needed)
+num_processes = len(configs)
```

Comment:
Not sure what you mean by "adjust as needed".

Reply:
Removed